zookeeper/src/main/java/com/linecorp/armeria/client/zookeeper/ZooKeeperEndpointGroup.java

```diff
@@ -67,58 +66,41 @@ public ZooKeeperEndpointGroup(String zkConnectionStr, String zNodePath, int sess
      */
     public ZooKeeperEndpointGroup(String zkConnectionStr, String zNodePath, int sessionTimeout,
                                   NodeValueCodec nodeValueCodec) {
```

Comment:
Could we also add the variants that accept `CuratorFramework` instead of the connection string?